src/standard/BaseStandardVerifier.sol

```diff
@@ -881,43 +888,24 @@ abstract contract BaseStandardVerifier {
                     sub(
                         p,
                         addmod(
-                            mulmod(mload(C_U_LOC), mload(Z_OMEGA_EVAL_LOC), p),
+                            uzomega,
                             addmod(
-                                sub(p, mload(R_ZERO_EVAL_LOC)), // Change owing to the simplified Plonk
-                                addmod(
-                                    mulmod(mload(C_V4_LOC), mload(SIGMA2_EVAL_LOC), p),
-                                    addmod(
-                                        mulmod(mload(C_V3_LOC), mload(SIGMA1_EVAL_LOC), p),
-                                        addmod(
-                                            mulmod(mload(C_V2_LOC), mload(W3_EVAL_LOC), p),
-                                            addmod(
-                                                mulmod(mload(C_V1_LOC), mload(W2_EVAL_LOC), p),
-                                                mulmod(mload(C_V0_LOC), mload(W1_EVAL_LOC), p),
-                                                p
-                                            ),
-                                            p
-                                        ),
-                                        p
-                                    ),
-                                    p
-                                ),
-                                p
+                                r0neg, addmod(v4sig2, addmod(v3sig1, addmod(v2w3, addmod(v1w2, v0w1, p), p), p), p), p
```

Comment:
Right, so now you can just read right to left and see pretty easily that we are adding up the terms computed above.